internal/cmd/keepassxctemplatefuncs_test.go

```diff
@@ -170,3 +182,47 @@ func TestKeepassxcTemplateFuncs(t *testing.T) {
 		})
 	}
 }
+
+func setupKeepassEntry(
+	t *testing.T,
+	command, tempDir, database, databasePassword, groupName, entryName, entryUsername, entryPassword, attachmentName, attachmentData string,
+) {
```

Comment:
nit: instead of passing lots of arguments with the same type, pass a struct with named fields. For an example, see https://github.com/twpayne/chezmoi/blob/94f00d11d79581d5bc755574861a22b7716d81b0/internal/chezmoi/sourcestate.go#L340-L366

Reply:
done